### DISCUSSION TITLE
We should add PDM to https://packaging.python.org/en/latest/tutorials/managing-dependencies/

### DISCUSSION THREAD

**Discussion by pawamoy** [pdm-project/pdm#830] [General]
As well as trampolim, for the sake of showing every deserving alternative :stuck_out_tongue:

**Comment by frostming** [comment]
Thanks! Can you make a PR there?

**Reply by pawamoy** [reply]
I guess so :smile:

**Reply by pawamoy** [reply]
Could you come up with a description? I don't feel confident enough to write one that concisely describes PDM in comparison to alternatives :sweat_smile:

**Reply by pawamoy** [reply]
> pdm for a modern Python package management tool supporting PEP 582 (no virtualenvs: installation of packages in a __pypackages__ folder, à la node_modules) and relying on standards such as PEP 517 and PEP 621.

Let see what they say.

**Reply by pawamoy** [reply]
https://github.com/pypa/packaging.python.org/pull/1036
